tests/src/test/scala/system/basic/WskActivationTests.scala

```diff
@@ -54,4 +57,24 @@ abstract class WskActivationTests extends TestHelpers with WskTestHelpers with J
       }, 10, Some(1.second))
     }
   }
+
+  it should "fetch result using activation result API" in withAssetCleaner(wskprops) { (wp, assetHelper) =>
+    val name = "hello"
+    val expectedResult = JsObject("payload" -> "hello, undefined!".toJson)
+
+    assetHelper.withCleaner(wsk.action, name) { (action, _) =>
+      action.create(name, Some(TestUtils.getTestActionFilename("hello.js")))
+    }
+
+    val run = wsk.action.invoke(name, blocking = true)
+
+    // Use withActivation() to reduce intermittent failures that may result from eventually consistent DBs
+    withActivation(wsk.activation, run) { activation =>
+      retry({
+        val result = wsk.activation.result(Some(activation.activationId)).stdout
+
+        result should include(expectedResult.compactPrint)
```

Comment:
My concern with doing a direct JsObject comparison is that we may extend this test with the CLI and the test will fail since the CLI output differs.

REST endpoint output:
```
{"statusCode":0,"result":{"payload":"hello, undefined!"}}
```

CLI output:
```
{"result":{"payload":"hello, undefined!"}}
```

Reply:
`wsk activation result` should always be `{"payload":"hello, undefined!"}` in this case independent of the source. 

also the cli is broken i'd say it made up a field :)

Reply:
I may have something wrong in my local deployment. Looking into it.

Reply:
REST endpoint:
```
{"result":{"payload":"hello, undefined!"},"success":true,"status":"success"}
```
CLI:
```
{"payload": "hello, undefined!"}
```

Reply:
Shall we test based on the implementaton of `BaseWsk` then? I dislike the string based matching as well, although it sounds wrong to do it this way 🤔

Reply:
Hmm, what do we think about the CLI returning the same result that the REST endpoint does?

Reply:
no i think the cli is doing the convenient thing here and its ok to diverge.

Reply:
As to making the CLI and REST responses appear the same, this is related to the HTTP status code when the action errors which we've not closed.